src/main/java/presenters/InputPresenter.java

```diff
@@ -0,0 +1,23 @@
+package presenters;
+
+import java.util.Scanner;
+
+/**
+ * This presenter class contain methods that collect raw user input from the
+ * Main Interactive Panel
+ * NOTE: Since we are at the terminal version, this class is minimally implemented.
+ */
+public class InputPresenter {
+
+    /**
+     * Asks for and collect raw input from the user
+     * when player chooses whether to invest / upgrade or ignore a property, if available.
+     */
+    public static String ownerChooseAtProperty(String verb, String propName, int currPrice) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println(verb + " " +  propName + " for " + currPrice + " TBucks? (Y/N)");
```

Comment:
Do we want to add some like, invalid inputs check here? 
For example, if the user types something like: 

`yyyyy`, we should probably prompt them and tell them that this is an invalid input. 
Maybe we can have something like a: 
```
while (True) {
    input = collect input;
    if (input != "Y" || input != "N") {
        sout("Please try again");
    }
}
```

Reply:
Because if we don't do this "validate input", we probably will send some random stuff into the other parts of the program. 

Also, it probably would be better if we can make it "Case insensitive", something like: 
```
if (input.toUpper() != "Y" or input.toUpper() != "N") {
    sout("Try again!");
}
```
Sorry if my code looks a bit Python-ish, I forgot the method to turn strings to all uppercase.

Reply:
Good suggestions. However, since the input part is separate from the part which identify what the input actually is, this might take a bit of work. I'll try.